### ISSUE TITLE
FE: Pagination Prev/Next links should scroll to the top of the page

### ISSUE THREAD

**Issue by randimays** [department-of-veterans-affairs/va.gov-cms#14737]
## Description
When using the Prev/Next links in a paginated list such as [this](https://www.va.gov/outreach-and-events/outreach-materials/), the screen should scroll to the top so the veteran can find their place at the top of the list on the next or previous page.

Currently, the behavior scrolls to the bottom of the page and the pagination numbers are at the top of the view.

The pagination behavior should match [Find a Form paging](https://www.va.gov/find-forms/?q=disability).

### User story
**AS A** Veteran
**I WANT** the page to scroll to the top when I use Prev/Next links in paginated lists
**SO THAT** I can easily find my place in the list on the next page

Slack thread for context: https://dsva.slack.com/archives/C52CL1PKQ/p1691683090666189

### Engineering notes / background
Need to comb through content-build to look for instances of pagination across the site and fix scroll behavior for each instance.

The ideal solution is to use `<va-pagination>` to fix this, if possible.

## Acceptance criteria
- [ ] All instances of pagination across VA.gov (under the PW purview) scroll to the top when Prev/Next links are used
- [ ] Any pagination defects that belong to other teams are ticketed as appropriate or passed on to that team somehow
- [ ] Accessibility review

>Consider
> - Design / Accessibility reviews
> - Collab cycle requirements
> - Device sizes (mobile first)
> - Documentation updates / Change management
< - [Content model documentation](https://prod.cms.va.gov/admin/structure/cm_document)
> - Testing notes

- [ ]

**Comment by timcosgrove** [comment]
👀

**Comment by laflannery** [comment]
Sorry @wesrowe I forgot about that! I am ok with not moving forward on this because of that knowledge. I actually have other tickets I was purposely not sending to PW for this same reason

**Comment by randimays** [comment]
@laflannery @wesrowe should we do a quick scan of other paginated apps to make sure this one is the only problematic case?

**Comment by randimays** [comment]
We did a review of all the apps using pagination and the only ones are Facilities-specific that is already ticketed ([defect here](https://github.com/department-of-veterans-affairs/va.gov-cms/issues/14755)). Closing this as a no-op.
